public/app/features/serviceaccounts/state/actions.ts

```diff
@@ -101,3 +107,57 @@ export function removeServiceAccount(serviceAccountId: number): ThunkResult<void
     dispatch(loadServiceAccounts());
   };
 }
+
+// search / filtering of serviceAccounts
+const getFilters = (filters: ServiceAccountFilter[]) => {
+  return filters
+    .map((filter) => {
+      if (Array.isArray(filter.value)) {
+        return filter.value.map((v) => `${filter.name}=${v.value}`).join('&');
+      }
+      return `${filter.name}=${filter.value}`;
+    })
+    .join('&');
+};
+
+export function fetchServiceAccounts(): ThunkResult<void> {
```

Comment:
@Clarity-89 somehow it seems that the page load of serviceaccounts, do not trigger this function.
Which I want it to.

In the users page of /admin/users <- the `fetchUsers` function gets triggered on first render.
Like `componentDidMount`, but I cannot seem to implement that in the component here that I am building

-- I would love some help here if possible

Reply:
I don't see this function is called on the page load? Also what's the difference between this and `loadServiceAccounts`?

Reply:
removed the `loadServiceAccounts` infavor of always calling `fetchServiceAccounts`